zeppelin-web/pom.xml

```diff
@@ -93,6 +93,7 @@
             <exclude>src/fonts/google-fonts.css</exclude>
             <exclude>bower.json</exclude>
             <exclude>package.json</exclude>
+            <exclude>yarn.lock</exclude>
```

Comment:
Since yarn is superfast, somepeople can use `yarn install` instead of `npm install` in local build (not maven build) and it will create `yarn.lock`. Thus I added those 2 lines not to be annoyed. (Yarn guys will remove the `yarn.lock` file everytime to commit..)

It's the same reason why `npm-debug.log` is added to gitignore.